main.tex

```diff
@@ -1781,49 +1774,23 @@
 
 \end{enumerate}
 
-\item  BEHAVOUR OF PERSONS
+\item  BEHAVIOUR OF PERSONS
 
 \begin{enumerate}[label=\arabic{enumi}.\arabic*.]
 
 \item  Behaviour of Owners, Occupiers and invitees on Common Property
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must take all reasonable steps to ensure that their invitees, do not behave in a manner likely to unreasonably interfere with the peaceful enjoyment of any other person entitled to use the Common Property.
-
-\item  An Owner or Occupier of a Lot must:
-
-\begin{enumerate}[label=(\alph*)]
+\item  An Owner or Occupier of a Lot must take all reasonable steps to ensure that their invitees do not behave in a manner likely to unreasonably interfere with the peaceful enjoyment of any other person entitled to use the Common Property.
 
-\item  when on Common Property or visible from Common Property be adequately clothed and not use language or behave in a manner likely to cause offence or embarrassment to any other person;
 \newpage
 
 
-
-\item  not hold parties or gatherings on the Common Property unless the Owners Corporation first consent in writing;
```

Comment:
How many people constitutes a "party"? How much fun can they have before it crosses the line of it being a "party"? Don't worry, that much fun would be hard with these rules!